project/AkkaBuild.scala

```diff
@@ -425,9 +425,10 @@ object AkkaBuild extends Build {
     id = "akka-http-marshallers-scala-experimental",
     base = file("akka-http-marshallers-scala"),
     settings = parentSettings ++ Seq(
-      version := streamAndHttpVersion
+      version := streamAndHttpVersion,
+      resolvers += "Typesafe Repo" at "http://repo.typesafe.com/typesafe/releases/"
```

Comment:
I tried to add play-json dependency without it and it failed. I may have missed something, but it only appears in the atmos project.